addons/mail/static/src/discuss/discuss.xml

```diff
@@ -6,7 +6,19 @@
         <div t-if="store.isSmall and store.discuss.activeTab === 'mailbox'" class="w-100 border-bottom" t-call="mail.Discuss.mobileTopbar" t-ref="mobileTopbar"/>
         <div t-if="thread and !(store.isSmall and ['chat', 'channel'].includes(store.discuss.activeTab))" class="o-mail-Discuss-content d-flex flex-column h-100 w-100 bg-view overflow-auto" t-ref="content">
             <div class="o-mail-Discuss-header px-3 bg-100 border-bottom d-flex flex-shrink-0" style="height:42px;">
-                <ThreadIcon className="'me-2 align-self-center'" thread="thread"/>
+                <div t-if="thread and thread.isChannelGroup and !props.public" class="o-mail-Discuss-thread-avatar position-relative align-self-center align-items-center mt-2 mb-2 me-1">
+                    <img class="o-mail-Discuss-threadAvatar rounded-circle" t-att-src="thread.imgUrl" alt="Thread Image"/>
```

Comment:
Remove `-threadAvatar` classname on img
And rename `-thread-avatar` to `threadAvatar`